src/Core/Components/Radio/FluentRadio.razor

```diff
@@ -0,0 +1,28 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@using System.Diagnostics
+@inherits FluentComponentBase
+@typeparam TValue
+@{
+    Debug.Assert(Context != null);
+}
+<FluentField ForId="@Id"
+             Class="@ClassValue"
+             Style="@StyleValue"
+             Label="@Label"
+             LabelPosition="@Components.LabelPosition.After"
+             LabelWidth="@LabelWidth"
+             IncludeInputSlot="false">
+    <LabelTemplate>@LabelTemplate</LabelTemplate>
+    <ChildContent>
+        <fluent-radio id="@Id"
+                      checked="@(Context.CurrentValue?.Equals(Value) == true ? GetToggledTrueValue() : null)"
```

Comment:
As I understand the web component, the `RadioGroup.Value` is used to check the right Radio. This property would only be used if a `FluentRadio` was used outside a `Group`, which is not possible with us.